sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -366,6 +366,86 @@ a compile error if `PropertyListT` does not contain a property with the
 Available only if `propertyT` is the property key class of a compile-time
 constant property.
 
+// --- ROW BREAK ---
```

Comment:
You will also have to add these operators to the section above that has the class skeleton.